autopilot/Wine-Reviews/Autopilot Does Text.ipynb

```diff
@@ -0,0 +1,399 @@
+      "text/plain": [
+       "<Figure size 2560x1440 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ut.get_roc_curve(predictions, candidates, no_cal_labels)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Best Candidate Name: tuning-job-1-0988ed90ee1f45209c-014-154208dc\n",
+      "Best Candidate Threshold (Operation Point): 0.17469505965709686\n",
+      "Best Candidate Recall: 0.9126047811238746\n"
```

Comment:
Can we add some conclusion on comparison between the two models (with and without California)?